### PR TITLE
Medical self tending change

### PR DESCRIPTION
**Pull request by PowerfulBacon**
## About The Pull Request

Changes medical self-tending effects:
- You can now heal yourself without the metabolic restriction in a short timespan.
- Every time you self-tend with bruise packs or ointment, you will recieve 60s of a debuffing status effect (which stacks the more you heald).
- Upon recieving the status effect for the first time, you will be paralysed for 3 seconds and recieve 50 stamina damage.
- While you have the status effect, you will take constant stamina damage up to 50 stamina damage.
- While you have the status effect, your movement speed will be considerably reduced.

Changes status effects:
- All status effects that last for a specific duration will now display the time remaining on them.

Fixes:
- Fixes being able to use medical items on unhurt limbs.

Note for review: I haven't added the status effect define since they are kinda stupid and only half-used anyway. If this is important, then I will change it.

image

## Why It's Good For The Game

I have never really liked how our current self-tend works. Whether or not you have metabolites in your system is completely opaque and it just feels frustrating to not be able to use the medical items that you have on you when you are injured.

This changes allows you the freedom to use as much self-tending items as you want, but with a cost that stacks the more you use. I expect that most people will self-tend anyway, but if you are in a combat pinch and have a lot of damage it might be worth handing off to someone else since the effects are designed to be pretty debilitating in combat.

I have seen people ask to be healed, but the effects of this change haven't been that impactful. I do think that this change may reverse it slightly, but at the same time I don't expect a significant change where there wasn't one to begin with. In a weird way, we may observe that as people find these basic items more capable at healing slight damage, they take them more and end up asking someone else to heal them to avoid the effects if they need to heal fast.

Since you now have the options, as someone in control of a fight like an antagonist, you can choose to hide and wait out the effects. As someone like security who gets hunted down, it may be more worthwhile to avoid the debuff.

Overall, actually having a choice between good and bad is a lot more interesting than just being hard-locked out of something that has no feedback to the end user. The previous effect was simply not fun.

## Testing Photographs and Procedure

Since this is what most of you care about, here is a video of the slowdown:

https://github.com/user-attachments/assets/cc701be5-fef8-49f7-b441-f47f9ecf71e0

Here is the actual testing:

image

image

Other status effects with an indicator.

image

Bleeding working as intended.

image

Exercise working as intended.

image

Slimeskin working as intended.

image

Fix working as intended

## Changelog
:cl:
balance: Removes the effects that makes you unable to self-tend shortly after self-tending with ointment and bruise packs.
balance: Adds a significant combat debuff when self-tending with ointment and bruise-packs.
/:cl:

### CODE REVIEW

**Comment by Rukofamicom** [comment]
Haven't looked at code yet so the best I can offer is a general balance approval for now.

**Comment by Nakalon** [comment]
This feels like it game-ifies this system a bit. And I'm all for game-ifying stuff. But I don't get what the self tend system thing aims to do, avoid people from self tending when doctors are present? If so I think there would be better ways!

**Comment by PowerfulBacon** [comment]
The self tending thing comes from a desire to stop players ignoring everyone else and healing themselves. In my opinion the current system is too opaque and is only slightly effective, so I'm hoping a more transparent system with more choices is just more fun to interact with even if its not quite as effective as the original.

**Comment by Nakalon** [comment]
> The self tending thing comes from a desire to stop players ignoring everyone else and healing themselves. In my opinion the current system is too opaque and is only slightly effective, so I'm hoping a more transparent system with more choices is just more fun to interact with even if its not quite as effective as the original.

Im sorry if I sounded crass or overly critical. I actually like this approach and agree with you. I stopped self healing because of the current system because I just don't understand it. This is much more visual. Althought it does its job I think your system may be a bit conveluted or unimmersive. To me it doesn't make much sense you get fatigued from applying ointments to yourself.

I think a more realistic solution would maybe be to replace publicly aviable medkits with versions that aren't as effective, and couldn't heal you to full, and only stocking medbay with the standard medicine. I think that would be more realistic, but maybe not as fun or game-like as this system.

In short your idea is probably the better option

**Comment by Rukofamicom** [comment]
You get fatigued because the ointments and packs are very painful

**Review by Rukofamicom — APPROVED** [review]
Should consider a maximum upper limit for the debuff, perhaps a 5 minute limit. It also might not be necessary. 
Code looks good in any case.
